pkg/api/addons.go

```diff
@@ -83,7 +83,7 @@ func (cs *ContainerService) setAddonsConfig(isUpdate bool) {
 				MemoryRequests: "10Mi",
 				CPULimits:      "50m",
 				MemoryLimits:   "10Mi",
-				Image:          "mcr.microsoft.com/k8s/flexvolume/blobfuse-flexvolume",
+				Image:          "mcr.microsoft.com/k8s/flexvolume/blobfuse-flexvolume:1.0.5",
```

Comment:
let's make sure these are added to the VHD script for pre-baking (packer/install-dependencies.sh). How are versions currently defined for these images?